src/com/form.js

```diff
@@ -103,6 +103,7 @@
 			
 			isSubmitting = true;
 			submitting(isSubmitting);
+			App.callback(options.post.prePost);
```

Comment:
I think `submit` or `submitting` would be a better name.... since we do not have the `postPost` option...